providers/pushpad/src/lib/pushpad.provider.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { PushpadPushProvider } from './pushpad.provider';
+
+test('should trigger pushpad library correctly', async () => {
+  const provider = new PushpadPushProvider({
+    apiKey: 'test-token',
+    appId: '012345',
+  });
+});
```

Comment:
Please add a test case where the response is mocked,
For reference, you can check other providers spec files